packages/requestNetwork.js/src/servicesExternal/erc20-service.ts

```diff
@@ -0,0 +1,169 @@
+import { Web3Single } from './web3-single';
+
+// @ts-ignore
+import * as Web3PromiEvent from 'web3-core-promievent';
+
+const ERC20 = require('../lib/ERC20.json');
+
+/**
+ * The Erc20Service class manage the ERC20 token
+ */
+export default class Erc20Service {
+    /**
+     * ERC20 contract's abi
+     */
+    public static abiERC20: any = ERC20.abi;
+
+    private static web3Single: Web3Single;
+
+    /**
+     * ERC20 contract's address
+     */
+    private addressERC20: any = ERC20.abi;
```

Comment:
Same value as abiERC20?
Also, the init value is overwritten in the contrsuctor

Reply:
good point : `private addressERC20: string;`